platform/wismote/dev/acc-sensor.c

```diff
@@ -52,37 +52,40 @@ activate(void)
   P2OUT |= 0x48;
 
 
-  /* stop converting immediately
+  /* stop converting immediately */
   ADC12CTL0 &= ~ENC;
   ADC12CTL1 &= ~CONSEQ_3;
 
   /* Configure ADC12_2 to sample channel 11 (voltage) and use
-  /* the Vref+ as reference (SREF_1) since it is a stable reference
+   * the Vref+ as reference (SREF_1) since it is a stable reference */
   ADC12MCTL2 = (INCH_4 + SREF_1);
   ADC12MCTL3 = (INCH_5 + SREF_1);
   ADC12MCTL4 = (INCH_6 + SREF_1);
-  /* internal temperature can be read as value(3)
+  /* internal temperature can be read as value(3) */
   ADC12MCTL5 = (INCH_10 + SREF_1);
 
   ADC12CTL1 |= CONSEQ_3;
   ADC12CTL0 |= ENC | ADC12SC;
 
-  /*  Irq_adc12_activate(&acc_sensor, 6, (INCH_11 + SREF_1)); */
+  Irq_adc12_activate(&acc_sensor, 6, (INCH_11 + SREF_1));
+  #endif
   active = 1;
 }
 /*---------------------------------------------------------------------------*/
 static void
 deactivate(void)
 {
-  /*  irq_adc12_deactivate(&acc_sensor, 6);
-      acc_value = 0;*/
+  #if 0
+  irq_adc12_deactivate(&acc_sensor, 6);
+  acc_value = 0;
+  #endif
   active = 0;
 }
 /*---------------------------------------------------------------------------*/
 static int
 value(int type)
 {
-/*
+  #if 0
```

Comment:
I changed the "disable code" style from `/* */` to `#if 0 #endif` to match the rest of the file.